FBSimulatorControl/Utility/FBSimDeviceWrapper.m

```diff
@@ -87,22 +88,23 @@ - (BOOL)installApplication:(NSURL *)appURL withOptions:(NSDictionary *)options e
 {
   NSAssert([NSThread isMainThread], @"Must be called from the main thread.");
 
-  NSError *__autoreleasing innerError = nil;
-  NSError *__autoreleasing *innerErrorPointer = &innerError;
+  NSError *innerError = nil;
   NSInvocation *invocation = [NSInvocation invocationWithMethodSignature:[self methodSignatureForSelector:@selector(installApplication:withOptions:error:)]];
   [invocation setTarget:self.simulator.device];
   [invocation setSelector:@selector(installApplication:withOptions:error:)];
   [invocation setArgument:&appURL atIndex:2];
   [invocation setArgument:&options atIndex:3];
-  [invocation setArgument:&innerErrorPointer atIndex:4];
-  error = innerErrorPointer;
+  [invocation setArgument:&innerError atIndex:4];
```

Comment:
Did you test it? It was crashing before that change with innerErrorPointer was made.